homeassistant/components/lookin/entity.py

```diff
@@ -124,3 +130,57 @@ def __init__(
             self._power_on_command = POWER_ON_CMD
         if POWER_OFF_CMD in self._function_names:
             self._power_off_command = POWER_OFF_CMD
+
+
+class LookinPowerPushRemoteEntity(LookinPowerEntity):
```

Comment:
Do we still need `LookinPowerEntity` and `LookinEntityMixIn` or can we merge them into their only children?

Reply:
Yes, it is for the unmerged `fan` platform.